src/app/core/browse/browse.service.ts

```diff
@@ -155,6 +145,9 @@ export class BrowseService {
           args.push(`page=${options.pagination.currentPage - 1}`);
           args.push(`size=${options.pagination.pageSize}`);
         }
+        if (isNotEmpty(options.startsWith)) {
+          args.push(`startsWith=${options.startsWith}`);
```

Comment:
I'm getting this strange behavior of having different rows vs results in facets. You can reproduce it by following this steps and use the Atmire's database - https://www.dropbox.com/s/ovqp394y3vofnwa/entities7-test-db.sql.gz?dl=1:

Access the browse by subject page:
`/browse/subject`
Filter the results by letter F:
`/browse/subject?startsWith=F`
Choose: Financial Therapy that has 8 results
`/browse/subject?startsWith=F&value=financial%20therapy`
You will only see 4 records but financial therapy has 8. This is because the parameter "startsWith=F" is assumed and it's used along with the filter subject. I think the parameter shouldn't be set on this request.
I excepted to see the same results as:
`/browse/subject?value=financial%20therapy`

Reply:
The startsWith parameter is not included in the links anymore when navigating browse entries.